modules/network/network-managers/connectivity-configurations/main.bicep

```
@@ -1,3 +1,7 @@
+metadata name = 'Network Manager Connectivity Configurations'
+metadata description = '''This module deploys a Network Manager Connectivity Configuration.
```

something's off with the description. Probably:
```suggestion
metadata description = '''This module deploys a Network Manager Connectivity Configuration.

Connectivity configurations define hub-and-spoke or mesh topologies applied to one or more network groups.'''
```